ma_gym/envs/traffic_junction/traffic_junction.py

```diff
@@ -74,12 +104,106 @@ def __init__(self, grid_shape=(14, 14), n_agents=10, n_max=10, arrive_prob=0.5,
             self._obs_high = np.tile(self._obs_high, self.n_agents)
             self._obs_low = np.tile(self._obs_low, self.n_agents)
         self.observation_space = MultiAgentObservationSpace([spaces.Box(self._obs_low, self._obs_high) for _ in range(self.n_agents)])
-        self.seed()
+
+        self.agents_colors = [(random.randrange(256), random.randrange(256), random.randrange(256)) for _ in range(self.n_agents)]
```

Comment:
The colors have to be fixed and pre-defined.

This is required so that we have the same colors across multiple instances of the game and even if the random seed is changed.

Reply:
Let's have a fixed set of colors for all 'n_max' agents.